### ISSUE TITLE
/cclickthrough - Clicking through entities / blocks

### ISSUE THREAD

**Issue by leumasme** [Earthcomputer/clientcommands#97]
/cclickthrough block|entity @e[..]|minecraft:... or something like that.
Couldnt come up with a better command name...
Would disable the hitbox for selected entities/blocks.
Idea came from sheering sheep and wishing to be able to click through the ones that have already been sheered, but would also be useful for other things. crender only seems to disable render and not the actual hitbox.

**Comment by xpple** [comment]
What about a flag to `/crender` instead to remove the hitbox? Or does this deserve its own command?

**Comment by Earthcomputer** [comment]
No, it's got nothing to do with rendering.
Also, note that I probably won't merge this one until 1.17, when there is a permissions system.

**Comment by barraIhsan** [comment]
> when there is a permissions system.

what is that?

**Comment by xpple** [comment]
The proposed opt-in and opt-out system required for certain QuiltMC mods.

**Comment by Earthcomputer** [comment]
It's not required for the mods, it's required for them to be advertised in Quilt community spaces. Otherwise you may see it as just a recommendation (which I will be following).

**Comment by xpple** [comment]
I am aware of that. I should have added: "for them to be eligible to be discussed in Quilt community spaces".

**Comment by spaceclouds42** [comment]
Another use for this would be clicking through grass/flowers when using a sword, so to add on, an optional `whileHolding <item>` param at the end would be nice

**Comment by xpple** [comment]
That gets very close to cheating...

**Comment by barraIhsan** [comment]
> That gets very close to cheating...

yeah, but iirc `cghostblock` kinda allow that. You can run `cghostblock set <x> <y> <z> air` at solid block, and click them through the block
